### PR TITLE
script.module.musicbrainz 0.6.0

### PR DESCRIPTION
**Pull request by marcelveldt**
module for the musicbrainz python bindings

### CODE REVIEW

File: script.module.musicbrainz/addon.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+       provider-name="marcelveldt">
+  <requires>
+  <import addon="xbmc.python" version="2.1.0"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary>Python bindings for Musicbrainz' NGS webservice</summary>
+    <description>Kodi module library for the Python bindings for Musicbrainz' NGS webservice developed by alastair.</description>
+    <license></license>
```
Comment (phil65) [inline script.module.musicbrainz/addon.xml]: please add the license.

File: script.module.musicbrainz/addon.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.musicbrainz"
+       name="Musicbrainz NGS webservice python bindings"
+       version="0.6.0"
+       provider-name="marcelveldt">
+  <requires>
+  <import addon="xbmc.python" version="2.1.0"/>
```
Comment (phil65) [inline script.module.musicbrainz/addon.xml]: jarvis = 2.24.0

**Comment by marcelveldt** [comment]
just updated the license and the python version. Never knew that about the python version numbers. Where can I lookup which version belongs to which Kodi version ?

**Comment by phil65** [comment]
http://kodi.wiki/view/addon.xml

**Comment by marcelveldt** [comment]
Oops, should have known that one ;-)
Thanks!

**Comment by phil65** [comment]
seems fine, thx. Please squash.

**Comment by marcelveldt** [comment]
commits are squashed now

**Pull request merged**